Assets/Scripts/UI/LoadingScreen.cs

```diff
@@ -0,0 +1,21 @@
+using Celeritas.Game;
+using UnityEngine;
+
+namespace Celeritas.UI
+{
+	public class LoadingScreen : Singleton<LoadingScreen>
+	{
+		[SerializeField]
+		private GameObject frame;
+
+		public static void Show()
```

Comment:
A short code documentation above for this public method. Method name is pretty intuitive, but just for clean documents.